soh/soh/OTRGlobals.cpp

```diff
@@ -289,6 +289,7 @@ extern "C" void VanillaItemTable_Init() {
         GET_ITEM(ITEM_BULLET_BAG_50, OBJECT_GI_DEKUPOUCH, GID_BULLET_BAG_50, 0x6C, 0x80, CHEST_ANIM_LONG, MOD_NONE, GI_BULLET_BAG_50),
         GET_ITEM_NONE,
         GET_ITEM_NONE,
+        GET_ITEM_NONE,
```

Comment:
just a note that these correspond to the following in the decomp vanilla item table (https://github.com/zeldaret/oot/blob/c0cf83a814d13a9a07a2ca260203134fc4c5b735/include/z64item.h#L437-L441)
```c
    /* 0x7B */ GI_BULLET_BAG_50,
    /* 0x7C */ GI_ICE_TRAP, // freezes link when opened from a chest
    /* 0x7D */ GI_TEXT_0, // no model appears over Link, shows text id 0 (pocket egg)
    /* 0x7E */ GI_MAX
```

Reply:
Actually before you merge let me add a comment to that last one explaining it's supposed to match with `GI_MAX`. People shouldn't be adding to that table (since separating new stuff from vanilla stuff was part of the point of re-working all this stuff), but if they are for whatever reason they need to know not to add new stuff before that last one.

Reply:
Done